app/src/main/java/de/westnordost/streetcomplete/quests/access_point_ref/AddAccessPointRef.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.access_point_ref
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddAccessPointRef : OsmFilterQuestType<AccessPointRefAnswer>() {
+
+    override val elementFilter = """
+        nodes with
+        (
+          (highway = emergency_access_point)
+          or
+          (emergency = access_point)
+        )
```

Comment:
```suggestion
        (
          highway = emergency_access_point
          or emergency = access_point
        )
```
no need for the additional brackets